docs/en/modules/wifi.md

```diff
@@ -1143,7 +1144,8 @@ Sets SSID and password in AP mode. Be sure to make the password at least 8 chara
 	- `save` save configuration to flash.
 		- `true` configuration **will** be retained through power cycle. (Default)
 		- `false` configuration **will not** be retained through power cycle.
-	- Event callbacks will only be available if `WIFI_SDK_EVENT_MONITOR_ENABLE` is enabled in `user_config.h`
+	- Event callbacks will only be available if `WIFI_SDK_EVENT_MONITOR_ENABLE` is uncommented in `user_config.h`
+		- Please note: To ensure all SoftAP events are handled at boot time, all relevant callbacks must be registered as early as possible in `init.lua` with either `wifi.ap.config()` or `wifi.eventmon.register()`.     
```

Comment:
One comment about this. The way this reads, the cb's are optional, which they are technically, but I feel that we should add a caveat to the effect that:

Application developers​ should not assume that the WiFi is connected at any time during initialization start-up.  This should be validated either by using a wifi event callback or of polling the status on a timer.

Or words to that effect.